action-bump-version/action.yaml

```diff
@@ -0,0 +1,75 @@
+name: Bump patch version
+description: |
+  Bump the current patch version in file VERSION
+  and open a new PR with the changes.
+  file VERSION is expected to the patch number
+  on a single line
+  ...
+
+inputs:
+  current-version:
+    required: true
+    description: the current version to bump
+  patch-version-identifier:
+    default: VERSION_PATCH
+    description: |
+      the name of the variable that sets the patch version in the ./VERSION file
+  branch:
+    required: true
+    description: branch name for the new PR to open
```

Comment:
do we need this one? we already provide the branch when running an action, so this is somewhat redundant (and thus error prone)